classes/cmake.yaml

```diff
@@ -1,11 +1,15 @@
 inherit: [cpackage, make, install]
 
-buildTools: [cmake]
-buildVars: [CC, CXX]
+buildTools: [cmake, target-toolchain]
+buildVars: [CC, CXX, BOB_HOST_PLATFORM,
+    CMAKE_GENERATOR, CMAKE_GENERATOR_PLATFORM, CMAKE_GENERATOR_TOOLSET,
+    CMAKE_TOOLCHAIN_FILE, CMAKE_BUILD_TYPE, BUILD_SHARED_LIBS]
 buildScript: |
     # Make sure CMake finds other stuff by its own logic too
     CMAKE_FIND_ROOT_PATH=
     for i in "${@:2}" ; do
+        # CMake needs win paths if we are in MSYS2
+        [[ "$BOB_HOST_PLATFORM" == "msys" ]] && i="$(cygpath -m $i)"
```

Comment:
Hmm, isn't that more dependent on the target platform / toolchain? It should still be possible to compile for msys2/mingw23/mingw64 which do not take windows paths I suppose.

Reply:
Ahh, correct, it doesn't depend directly on MSYS. It's the combination of MSYS and Windows. I have to use the CMAKE_GENERATOR variable. If we use Visual Studio, we are sure, that we are on Windows system. I guess the unix paths should work, if we build e.g. with Unix Makefiles generator.